nexus/src/lib.rs

```diff
@@ -351,6 +353,17 @@ impl nexus_test_interface::NexusServer for Server {
             .unwrap();
     }
 
+    async fn inventory_collect_and_get_latest_collection(
```

Comment:
This API is part of the `nexus_test_interface` API, and is basically punched through so tests which create synthetic disks can also "cause inventory collection to happen". They need it, to get the `inv_zpool` table populated.